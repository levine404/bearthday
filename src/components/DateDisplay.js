import React from 'react';

const DateDisplay = ({ label, date, changeDate }) => {
  return (
    <input
      type="number"
      value={date}
      placeholder={label}
      onChange={changeDate}
    />
  );
};

export default DateDisplay;
