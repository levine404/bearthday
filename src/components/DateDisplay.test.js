import React from 'react';
import { shallow } from 'enzyme';
import DateDisplay from './DateDisplay';

describe('DateDisplay', () => {
  it('renders', () => {
    const D = new Date('2016-1-3');
    const wrapper = shallow(<DateDisplay label="test" date={D} changeDate={() => {}} />);

    expect(wrapper).toMatchSnapshot();
  });
});