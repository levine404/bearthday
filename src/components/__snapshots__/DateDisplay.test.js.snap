// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateDisplay renders 1`] = `
<input
  onChange={[Function]}
  placeholder="test"
  type="number"
  value={2016-01-03T05:00:00.000Z}
/>
`;
